src/prefect/tasks/core/collections.py

```diff
@@ -7,6 +7,7 @@
 """
 from typing import Any, Iterable
 
+import prefect
```

Comment:
Following this, could you remove the quotations around the type definition [here](https://github.com/PrefectHQ/prefect/blob/18f61a0a878320d7fb977c18f27393916f03990a/src/prefect/tasks/core/collections.py#L33)